test/fuzz/gen.ml

```diff
@@ -267,7 +267,9 @@ let func env =
   Env.refill_fuel env;
   let* locals = list (local env) in
   let* type_f = B.block_type env in
+  let _ = Env.add_block env type_f Env.Func in
```

Comment:
Please don't use `let _`. :-)

Either `let () = ...` or `let (_meaningful_name : with_the_type) = ...` depending on if it's unit or not.

It will avoid some trouble if you ever change the arity of `Env.add_block` and then only partially apply the function.